readme.md

```diff
@@ -0,0 +1,14 @@
+# Introduction
+
+This is library that lists all of the silly phrases used for the [typedigital](https://typedigital.de/) website. 
+Upon clicking on the company logo found in the top left corner it will change to a combination of two words beginning with T and D respectively. E.g. typedigital -> thatdoge. It has been launched externally so that anyone with a funny idea can add that to the website by contributing to this small project.
+
+# Contributing
+
+If you have an idea that you want to add to our website feel free to simply open up a pull request and add it to the already existing list. 
+The only rules you have to follow are the following: 
+
+ - adhere to the format: 2 words, first one beginning with T, second one with D
+ - don't defame, discirmiate or hurt anyone. These expressions are supposed to be fun and not disrespectful. Feel free to meme like us.
+
+This Project is released under the unilicence and we treat every contribution as such.  
```

Comment:
small typo in "unlicense" :D 
everything else is pretty cool.

I would insert an own headline for the `License` with the last line.